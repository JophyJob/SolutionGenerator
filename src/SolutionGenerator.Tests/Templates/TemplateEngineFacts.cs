// --------------------------------------------------------------------------------------------------------------------
// <copyright file="TemplateEngineFacts.cs" company="WildGums">
//   Copyright (c) 2012 - 2016 WildGums. All rights reserved.
// </copyright>
// --------------------------------------------------------------------------------------------------------------------

namespace SolutionGenerator.Tests.Templates
{
    public class TemplateEngineFacts
    {
        
    }
}