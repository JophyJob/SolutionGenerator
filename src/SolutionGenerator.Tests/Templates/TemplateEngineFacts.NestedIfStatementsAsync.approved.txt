// --------------------------------------------------------------------------------------------------------------------
// <copyright file="MainViewModel.cs" company="My company">
//   Copyright (c) 2008 - 2017 My company. All rights reserved.
// </copyright>
// --------------------------------------------------------------------------------------------------------------------


namespace SomeSolution.ViewModels
{
    using System;
    using System.Threading.Tasks;
    using Catel;
    using Catel.Data;
    using Catel.MVVM;
    using Catel.Reflection;

    public class MainViewModel : ViewModelBase
    {
        public MainViewModel()
        {
            Title = "SomeSolution example";
        }

        
        public class AdvancedSolution
        {
            
            public const bool IsSupportAdvanced = true;
            
        }
        
    }
}